documentation/index.html

```diff
@@ -425,6 +425,230 @@ <h3 class="content-heading">Getting Started</h3>
                     </pre>
                 </div>
                 <hr>
+                <!-- image component -->
+                <div class="image-container">
+                    <p class="content-sub_heading">Images</p>
+                    <p class="sub-heading-description">The Image component is used to display images</p>
+                    <p class="content-sub_heading-sm">Responsive Image</p>
+                    <p class="sub-heading-description">Use w-size and h-size in the class name and give the required width. For rounded image use "img-rounded" in the class name</p>
+                    <div class="display-image">
+                        <img src="../assets/avatar.png" alt="" class="w-10 h-10">
+                        <img src="../assets/avatar.png" alt="" class="img-rounded w-10 h-10">
+                    </div>
+                    <div class="code-snippet">
+                        Use the code snippets mentioned below:
+                        <pre id="myInput">
+                            <code class="language-html">&lt;img src=&quot;../assets/avatar.png&quot; alt=&quot;&quot; class=&quot;w-10 h-10&quot;&gt;
+                                
+&lt;img src=&quot;../assets/avatar.png&quot; alt=&quot;&quot; class=&quot;img-rounded w-10 h-10&quot;&gt;</code>
+                        </pre>
+                    </div>
+                </div>
+                <hr>
+                <!-- input container -->
+                <div class="input-container">
+                    <div class="content-sub_heading">Input</div>
+                    <p class="sub-heading-description">The input component is a component that is used to get user input in a text field.</p>
```

Comment:
input html code added in this branch there is a separate branch for it. Also you HTML is too huge maintaining it would be troublesome. Make separate HTML files for each component would be my suggestion. mostly others are doing that only

Reply:
At this point, if I try to make separate HTML file, it would be a quite challenging task. So, I think I should continue doing it this way only. And I will keep this thing in mind from next time!